docs/guides/set-up-webhooks.md

```diff
@@ -0,0 +1,153 @@
+# Webhooks for flexible scheduling notifications
+Webhooks are the easiest and fastest solution for quick notification. If something happens, they send the automated message to particular URL. It works for notification and also integration with different platforms. So, how does it work in Vyte?
+
+## About our hooks
+The hook has two main properties: `target_url` and `action`. Target URL leads to the recipient application (your website or app), `action` defines the type of event happened. For now Vyte uses several hooks:
+
+* Event:
+    created
+    confirmed
+    updated
+    cancelled
+    voted
+    new_invitee
+
+* User:
+    joined
+
+* Vyteme:
+    updated
```

Comment:
you should explain what each does (it's probably already in the API ref just reuse it)

Reply:
added the link 04ac6d4c68efa1e05553386bbc2d8e0c28248c37